cmd/serve.go

```diff
@@ -268,12 +269,11 @@ func serveAPIService(config config.Configuration, wg *sync.WaitGroup) {
 	db.Init()
 
 	service := &apiservice_service.APIService{
-		Config:         config,
-		Version:        serverVersion,
-		Database:       db,
-		TimeNow:        time.Now,
-		Log:            log,
-		AlertSvcClient: alertservice_client.NewClient(config.AlertService),
```

Comment:
Why have you removed AlertSvcClient: alertservice_client.NewClient(config.AlertService)?